README.rst

```diff
@@ -2,6 +2,7 @@ Torch autodiff for DFT-D3
 =========================
 
 .. image:: https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10-blue
+   :target: https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10-blue
```

Comment:
This link target is not useful

Reply:
Anything useful to put there? I only used that dummy, because without the `target`, the other badges were pushed to a new line.

Reply:
On shields IO this information is pulled from PyPI rather than created manually, so linking to PyPI seems like an idea.

```rst
.. image:: https://img.shields.io/pypi/pyversions/tad_dftd3
   :alt: PyPI - Python Version
```

Reply:
Seems like this information is missing. I guess, we need to specify the Python versions in `setup.cfg`.

However, without the `target` directive, the weird formatting still remains (see README on main branch).

Reply:
I think we have enough badges and don't really need this one.